tensorflow/lite/kernels/batch_matmul_test.cc

```diff
@@ -483,7 +483,12 @@ class QuantizedBatchMatMulOpModel : public SingleOpModel {
     input_size_ = total_input_size / batches_;
 
     lhs_id_ = AddInput(lhs);
-    rhs_id_ = AddInput({lhs.type, {input_size_, units_}, lhs.min, lhs.max});
+    rhs_id_ = AddInput({lhs.type,
+                        {input_size_, units_},
+                        0,
+                        0,
```

Comment:
The `lhs_id_ = AddInput(lhs);` line will create an input tensor with `lhs_id_` and set the scaling and zero-point based either on the min/max range of the parameter `t` or on the provided scaling and zero_point of parameter `t` if the range is left to 0 (see [SingleOpModel::AddTensor](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/test_util.h#L626)). The change I did on the creation of the `rhs_id_` input allows to directly recuperate the calculated scaling and zero-point of `lhs_id_` without needing any recalculation if a min/max range is used and is also more general as it allows the `lhs` parameter of `QuantizedBatchMatMulOpModel` to either have a min/max range or an explicit scaling and zero point as done in `SimpleTestQuantizedInt16`.

I set the input scale and zero-point manually in `SimpleTestQuantizedInt16` instead of letting `SingleOpModel::AddTensor` do it based on the range to ensure that the zero-point is null (int16 tensors are restricted to null zero-points). The problem is that even if I pass a symmetric range like [-10, 10], the scaling is calculated on the  asymmetric [2^15, 2^15 - 1] range and the zero-point will thus be -1. If needed I could change the test a bit to use a kMin/kMax multiplication factor to nudge a bit the range similar to what was done for some other int16 tests.

Reply:
Yep, I see what you are saying. Looks good! 👍